### DISCUSSION TITLE
UI Color Palette Idea

### DISCUSSION THREAD

**Discussion by e-laflamme** [CSC207-2022F-UofT/course-project-my-dog-ate-my-hw#31] [Ideas]
I was thinking about what colors the main UI of the app should use and I found this palette:

<img width="1396" alt="Screen Shot 2022-11-17 at 1 48 13 PM" src="https://user-images.githubusercontent.com/113330224/202532594-3f95e519-8b2b-449b-abce-b7281e67707c.png">

I thought the colors look nice and soft, good for a pet and study app. What does everyone think about it? Does anyone have some other ideas of palettes?

**Comment by stephaniesaab** [comment]
I agree with this colour palette, the variety also makes it possible to distinguish colours for various types of colour-blindness. The colours should reflect the state of the pet so for the 'happy pet' (when it has good health and customizations) it can have these colours, then when it loses health it can change to darker, more pronounced colours. Of course, this is just an idea for extension. Ultimately, the colours shouldn't be hard-coded in and it would be best to use some easily-changed variables (static final) so that they can be changed later if desired.
